### PR TITLE
core: commander: use 'reboot' instead of 'shutdown -r' to reboot

### PR DESCRIPTION
**Pull request by Williangalvani**
fixes an issue where the pi is shutting down instead of rebooting.

image will be up at williangalvani/blueos-core:reboot when CI finishes.

### CODE REVIEW

File: core/services/commander/main.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ async def shutdown(response: Response, shutdown_type: ShutdownType, i_know_what_
             output = run_command(f"(sleep {hold_time_seconds}; sudo shutdown --reboot -h now)&")
             logger.debug(f"reboot: {output}")
         elif shutdown_type == ShutdownType.POWEROFF:
-            output = run_command(f"(sleep {hold_time_seconds}; sudo shutdown --poweroff -h now)&")
+            output = run_command(f"(sleep {hold_time_seconds}; sudo reboot)&")
```
Comment (patrickelectric) [inline core/services/commander/main.py:118]: This is replacing power off and not rebooting, is this correct ?

Comment (Williangalvani) [reply core/services/commander/main.py:118]: lol it is not

**Review by patrickelectric — APPROVED** [review]

**Review by ES-Alexander — APPROVED** [review]
Assuming this has been tested and confirmed to solve the issue, seems fine :-)

**Comment by Williangalvani** [comment]
tested and it works.

**Pull request merged**